src/registrar/views/portfolios.py

```diff
@@ -41,6 +42,29 @@ def get(self, request):
         return render(request, "portfolio_requests.html")
 
 
+class PortfolioMembersView(PortfolioMembersPermissionView, View):
+
+    template_name = "portfolio_members.html"
+
+    def get(self, request):
+        """Add additional context data to the template."""
+
+        if self.request.user.is_authenticated:
+            request.session["new_request"] = True
+
+        # We can override the base class. This view only needs this item.
+        context = {}
+        portfolio = self.request.session.get("portfolio")
+        if portfolio:
+            # ------- Gets all members
+            member_ids = UserPortfolioPermission.objects.filter(portfolio=portfolio).values_list("user__id", flat=True)
+
+            all_members = User.objects.filter(id__in=member_ids)
+            context["portfolio_members"] = all_members
+            context["portfolio_members_count"] = all_members.count()
```

Comment:
Remove this context. Even if we end up needing something like this for an export button, this won't be the solution as the table content is dynamic; this means that you can have a count > 0, delete a member on the table, but the button will still be visible because the context test has already passed.